yaksh/live_server_tests/load_test.py

```diff
@@ -31,8 +31,11 @@ def setUpClass(cls):
         app_label = 'yaksh'
         group_name = 'moderator'
         cls.group = create_group(group_name, app_label)
```

Comment:
You can create moderator group and add to it instead of using create_group method.

Reply:
https://github.com/FOSSEE/online_test/blob/519cbb803c8020527578d2e9cc1b55a20f29c8ee/yaksh/live_server_tests/load_test.py#L33 
I have picked this way from the original repo, by using this method I don't have to rewrite all the code to give permissions.

Reply:
It should not be dependent on a management method. If the method is removed it will fail the tests unnecessarily. Better is it should be independent